vendor/bat-native-ads/src/bat/ads/internal/unittest_util.cc

```diff
@@ -258,8 +253,24 @@ void MockSetUint64Pref(const std::unique_ptr<AdsClientMock>& mock) {
 
 void MockClearPref(const std::unique_ptr<AdsClientMock>& mock) {
   ON_CALL(*mock, ClearPref(_))
-      .WillByDefault(
-          Invoke([](const std::string& path) { g_prefs.erase(path); }));
+      .WillByDefault(Invoke([](const std::string& path) {
+        const std::string pref_path = GetUuidForCurrentTest(path);
+        g_prefs.erase(pref_path);
+      }));
+}
+
+void MockHasPrefPath(const std::unique_ptr<AdsClientMock>& mock) {
```

Comment:
is this never called?

Reply:
@moritzhaller called from `unittest_base.cc` in `void MockPrefs(const std::unique_ptr<AdsClientMock>& mock)`